src/components/buttonsIndicators/buttonCta/style.ts

```diff
@@ -76,8 +78,10 @@ export const StyledContent = styled.div`
   ${(p: Props) => generateContent(p.color, (p.disabled || false))};
 ` as any
 
-export const StyledIcon = styled.span`
-  display: inline-block;
+export const StyledIcon = styled.div`
+  display: block;
   vertical-align: bottom;
+  height: '14px';
+  color: blue;
```

Comment:
Is this like default fallback if color is not provided from the parent wrapper?

Reply:
Er, no. I shouldn’t have edited this file. 

...I did a hasty push before finishing for the day...